src/decorators/auth0authenticateFactory.js

```diff
@@ -53,7 +53,7 @@ module.exports = (domain, clientId, clientSecret, audience, cache) => {
       log('Token does not exist')
       return false
     }
-    if (!token.access_token || !token.access_token || !token.expiresAfter) {
```

Comment:
I found this "bug" ... and removed the unneeded extra '!token.access_token' check.